README.rst

```diff
@@ -273,20 +273,20 @@ Adding and removing keys
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 When creating new files, ``sops`` uses the PGP, KMS and GCP KMS defined in the
-command line arguments ``--kms``, ``--pgp``, ``--gcp-kms`` or ``--azure-kv``, or from
-the environment variables ``SOPS_KMS_ARN``, ``SOPS_PGP_FP``, ``SOPS_GCP_KMS_IDS``,
-``SOPS_AZURE_KEYVAULT_URL``. That information is stored in the file under the
-``sops`` section, such that decrypting files does not require providing those
-parameters again.
+command line arguments ``--kms``, ``--aws-profile``, ``--pgp``, ``--gcp-kms`` or
```

Comment:
I don't think `--aws-profile` needs to be specified here, just as we don't specify `--encryption-context`